### PR TITLE
Clear localStorage if user logout and improving get token

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

**Comment by ghost** [comment]
:ok:

**Comment by krzysztofzygmunt** [comment]
:ok:

File: ui/static/app/http-client.ts [hunk 1]
```diff
@@ -12,14 +12,12 @@ export class HttpClient {
...
-  ) {
-    this.apiToken = this.authService.getToken();
-  }
+  ) { }
 
   getAuthorizationHeader(): Headers {
     let headers: Headers = new Headers();
     headers.append("Content-Type", "application/json");
-    headers.append("Authorization", `Token ${this.apiToken}`);
+    headers.append("Authorization", `Token ${this.authService.getToken()}`);
```
Comment (mkurek) [inline ui/static/app/http-client.ts:20]: Please no. In this case, before every requesta to api there will be additional request for user token, am i right?

Comment (szok) [inline ui/static/app/http-client.ts:20]: No, its ok, look:
https://github.com/allegro/django-powerdns-dnssec/blob/develop/ui/static/app/auth/auth.service.ts#L43

Token is getting only one in login page. 
Earlier token was taken from localStorage only when in create object from class.
Now is always get from  localStorage.

**Pull request merged**